libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -346,7 +346,7 @@ void AP_CRSF_Telem::calc_battery()
 
     _telem.bcast.battery.remaining = _battery.capacity_remaining_pct(0);
 
-    int32_t capacity = _battery.pack_capacity_mah(0);
+    const int32_t capacity = used_mah;
```

Comment:
This change looks unrelated to the topic of the PR.

Reply:
Changed the heading